exercises/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,40 @@
+// Exercise 1 (commented out to see the result of exercise 2)
+/*
+function percentage(num, per){
+    return Math.round((num / (num + per)) * 100) + "%";
+}
+
+function print(students, mentors){
+   var percentageS = percentage(students, mentors);
+   var percentageM = percentage(mentors, students);
+   return "Percentage students: " + percentageS;
+   return "Percentage mentors: " + percentageM;
+}
+
+console.log(print(15, 8));
+console.log(print(8, 15));
+
+*/
+//Exercise 2
+
 var mentor1 = "Daniel";
 var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+var mentor5 = "Yohannes"
```

Comment:
Be careful of doing stuff like this. You'e took the semi colon away by accident.